processes/processes_test.go

```diff
@@ -179,63 +183,176 @@ func TestCollectMetrics(t *testing.T) {
 			mc := &mcMock{}
 			procPlugin.mc = mc
 
-			mc.On("GetStats").Return(map[string][]Proc{
-				"NetworkManager": []Proc{mockProc},
-				"fake":           []Proc{mockProc2},
+			mc.On("GetStats").Return(map[string]map[int]Proc{
+				"NetworkManager": map[int]Proc{
+					mockProcPid: mockProc,
+				},
+				"fake": map[int]Proc{
+					mockProcPid2: mockProc2,
+					mockProcPid3: mockProc3,
+				},
 			}, nil)
 
 			Convey("when names of collect metrics are valid", func() {
 				results, err := procPlugin.CollectMetrics(mockMts)
 
 				So(err, ShouldBeNil)
-				So(len(results), ShouldEqual, len(mockMts))
+				So(len(results), ShouldEqual, 12)
```

Comment:
as above - if you use some count explicitly, please add a comment at least, even in tests